docs/guides/how_to_guides/creating_batches/how_to_create_a_batch_request_using_an_active_data_connector.rst

```diff
@@ -0,0 +1,269 @@
+          "name": "a"
+            }
+          }
+        )
+
+
+7. **Construct a Validator**
+
+  The ``BatchRequest`` and ExpectationSuite can be used to create a Validator.
+
+  .. code-block:: python
+
+    my_validator = context.get_validator(
+      batch_request=batch_request,
+      expectation_suite=suite
+    )
+
+
+8. **Check your Validator**
+
+  You can check to see if the correct Batch was retrieved by checking the ``active_batch``'s ``batch_definition``.
+
+  .. code-block:: python
+
+    my_validator.active_batch.batch_definition
+
+  The expected output should show ``batch_identifiers`` corresponding to ``a-1.csv`` namely ``"{'name': 'a', 'num': '1'}"}``
+
+  .. code-block:: python
+
+    {'datasource_name': 'mydatasource', 'data_connector_name': 'my_data_connector', 'data_asset_name': 'my_reports', 'batch_identifiers': "{'name': 'a', 'num': '1'}"}
+
+
+  You can also check that the first few lines of your Batch are what you expect by running:
+
+  .. code-block:: python
+
+    my_validator.active_batch.head()
+
+
+  Now that you have a Validator, you can use it to create Expectations or validate the data.
+
+
+Additional Notes
+----------------
+
+``BatchRequest`` can also support ``index`` and ``limit`` in the ``data_connector_query``.
+
+  Using the same ``reports`` directory as above:
+
+  .. code-block:: bash
+
+    reports/a-1.csv
+    reports/b-2.csv
+    reports/c-3.csv
+    reports/d-4.csv
+    reports/e-5.csv
+
+
+  The ``BatchRequest`` can retrieve Batches by ``index``. The following examples retrieve the first (``index = 0``)
+
+  .. code-block:: python
+
+    batch_request = BatchRequest(
+      datasource_name="mydatasource",
+      data_connector_name="my_data_connector",
+      data_asset_name="my_reports",
+      data_connector_query={
+          "index": 0
+          }
+        )
+
+    my_validator = context.get_validator(
+      batch_request=batch_request,
+      expectation_suite=suite
+    )
+    print(my_validator.active_batch.batch_definition)
+
+    # batch corresponding to a-1.csv
+    {'datasource_name': 'mydatasource', 'data_connector_name': 'my_data_connector', 'data_asset_name': 'my_reports', 'batch_identifiers': "{'name': 'a', 'num': '1'}"}
+
+  last (``index=-1``) batches.
+
+  .. code-block:: python
+
+    batch_request = BatchRequest(
+      datasource_name="mydatasource",
+      data_connector_name="my_data_connector",
+      data_asset_name="my_reports",
+      data_connector_query={
+          "index": -1
+          }
+        )
+
+    my_validator = context.get_validator(
+      batch_request=batch_request,
+      expectation_suite=suite
+    )
+
+    print(my_validator.active_batch.batch_definition)
+
+    # batch corresponding to e-5.csv
+    {'datasource_name': 'mydatasource', 'data_connector_name': 'my_data_connector', 'data_asset_name': 'my_reports', 'batch_identifiers': "{'name': 'e', 'num': '5'}"}
+
+
+  The ``BatchRequest`` can also be used to retrieve Batches by ``limit``.
```

Comment:
What purpose does this serve until multi-batch is a thing? Is this equivalent to `"index": 0`?